pkg/kubelet/status/state/state_checkpoint.go

```diff
@@ -76,10 +76,14 @@ func (sc *stateCheckpoint) restoreState() error {
 	if err != nil {
 		return fmt.Errorf("failed to get pod resource allocation info: %w", err)
 	}
-	err = sc.cache.SetPodResourceAllocation(praInfo.AllocationEntries)
-	if err != nil {
-		return fmt.Errorf("failed to set pod resource allocation: %w", err)
+
```

Comment:
This was only necessary to avoid breaking it. In this [PR](https://github.com/kubernetes/kubernetes/pull/129504) this call **SetPodResourceAllocation** is removed from here.